docs/pages/includes/desktop-access/desktop-config.yaml

```diff
@@ -61,4 +61,9 @@ windows_desktop_service:
       environment: dev
   - match: '^.*\.prod\.example\.com$'
     labels:
-      environment: prod
\ No newline at end of file
+      environment: prod
+
+  # Labels to attach to the Windows Desktop Service. This is used internally,
+  # any custom labels added won't affect the Windows hosts.
```

Comment:
I'm not sure what the "This is used internally" sentence means. Since this field is only used internally, when would someone want to set a custom value for it?

Reply:
They shouldn't, but I didn't know if we allowed for config properties that are only used internally to go undocumented on the config reference page.